project/Workflows.scala

```diff
@@ -0,0 +1,117 @@
+import sbt._
+import sbtghactions.GenerativePlugin.autoImport._
+import CoursierWorkflow.autoImport._
+import ReleaseWorkflow.autoImport._
+
+object Workflows extends AutoPlugin {
+  override def requires: Plugins = ReleaseWorkflow && CoursierWorkflow
+
+  override def buildSettings: Seq[Def.Setting[_]] = Seq(
+    githubWorkflowGeneratedCI := {
+      def testSteps(tag: String, sbtTestSteps: WorkflowStep*): List[WorkflowStep] =
+        WorkflowStep.Checkout ::
+          coursierSetup.value ::
+          coursierCache.value ::
+          checkWorkflows.value ::
+          Sbtn(
+            name = Some("Checks"),
+            commands = List(
+              "scalafmtSbtCheck",
+              "scalafmtCheckAll",
+              "compile",
+              "doc",
+              "docs/mdoc",
+              "scalafix --check"
+            ).mkString("; ") :: Nil
```

Comment:
To run them in parallel (respecting inter-dependencies):
```scala
.mkString("all", "; ")
```

Not sure, but `all` might not work with `scalafix --check`, so it might be useful to define a one-word alias for it.

Reply:
Not sure if this is the reason, but without `all` it seems to run same or even faster. 
Does it work with `sbtn`?
Maybe it's just randomness of node load, but [here](https://github.com/fs2-blobstore/fs2-blobstore/actions/runs/391689065) for example `checks` finished in ~1m30s. While [here](https://github.com/fs2-blobstore/fs2-blobstore/actions/runs/393813251) with `all` in ~1m45s.

Reply:
The time in sbt seems to be almost the same: [76s](https://github.com/fs2-blobstore/fs2-blobstore/runs/1474111577?check_suite_focus=true#step:6:96) vs. [70s](https://github.com/fs2-blobstore/fs2-blobstore/runs/1479892016?check_suite_focus=true#step:5:97). So feel free to disregard this.